src/User.hpp

```diff
@@ -6,13 +6,14 @@
 class User
 {
 public:
-    User(const std::string & name,
-         const std::string & nick,
-         const std::string & group,
-         const std::string & gitHub,
-         const std::string & firecode);
+    User(const std::string name,
+         const std::string nick,
+         const std::string group,
+         const std::string gitHub,
+         const std::string firecode);
```

Comment:
why not references? std::string_view can be used as well

Reply:
My mistake